process/block/preprocess/basePreProcess.go

```diff
@@ -1,6 +1,8 @@
 package preprocess
```

Comment:
This has only 56.7% code cov. SmartContractResults and Transactions are below 40%.

Reply:
run together with blockprocessor. :)